index.html

```diff
@@ -0,0 +1,931 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Maven+Pro&display=swap" rel="stylesheet"> 
+    <link href="//db.onlinewebfonts.com/c/3f908bf27ca39531334bb55a36cbf143?family=Avalon+Medium" rel="stylesheet" type="text/css"/>
+    <title>TNW</title>
+</head>
+<body>
+    <div class="upper-nav navSwitch">
+        <ul>
+            <li><a href="#">News</a></li>
+            <li><a href="#">TNW2020</a></li>
+            <li><a href="#">Business</a></li>
+            <li><a href="#">AMAs</a></li>
+            <li><a href="#">Spaces</a></li>
+            <li><a href="#">Terms & Conditions</a></li>
+        </ul>
+        <ul>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Advertisement</a></li>
+            <li><a href="#">Jobs</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+        </div>
+    <nav>
+        <div class="lower-nav">
+            <img src="https://smart.beamian.com/wp-content/uploads/2018/05/the_next_web_logo.jpg" alt="TNW_Logo"/>
+            <div class="adjust-flex">
+            <ul class="navSwitch">
```

Comment:
The logo shouldn't appear on large screens.

Reply:
The Logo appears on the original Page on the Large Screen. It appears when you scroll down a few lines. I tried to find how to show it on scroll but couldn't find a way to do it without Javascript.